dsl/camel-jbang/camel-jbang-core/pom.xml

```diff
@@ -136,6 +141,11 @@
             <version>${jansi-version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.codehaus.plexus</groupId>
+            <artifactId>plexus-xml</artifactId>
+        </dependency>
```

Comment:
What is this used for ?

Reply:
This is slightly unrelated. The latest maven 4 builds include a fix in the way transitive dependency management is computed. This dependency is actually included in the dep tree with maven 3, but not with Maven 4, so I just included it at the same time. But this could be splitter into a different PR if needed.